blog/trackers-who-steal.md

```diff
@@ -0,0 +1,1225 @@
+- Their German and Australian sites appear to have been affected earlier, with our first observations on 10th December 2017 for ticketmaster.de and 20th December for ticketmaster.com.au. These sites were fixed at the same time as the others.
+
+In their [disclosure](https://security.ticketmaster.co.uk/), Ticketmaster say that UK customers were affected between February and June 23rd, and international customers could have been affected from September 2017. Again this matches up with our data, though we have no observations for international sites before December 2017.
+
+Our data also shows several other sites affected by this attack:
+
+<table class="table table-hover">
+  <thead>
+    <tr>
+      <th>Site</th>
+      <th>Affected from</th>
+      <th>Affected to</th>
+      <th>Pages</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>otel.com</td>
+      <td>10/12/2017</td>
+      <td>21/06/2018</td>
+      <td>125</td>
+    </tr>
+    <tr>
+      <td>www.cheaperthandirt.com</td>
+      <td>19/01/2018</td>
+      <td>03/06/2018</td>
+      <td>42</td>
+    </tr>
+    <tr>
+      <td>www.printninja.com</td>
+      <td>16/02/2018</td>
+      <td>20/11/2018</td>
+      <td>45</td>
+    </tr>
+    <tr>
+      <td>www.vitacost.com</td>
+      <td>26/02/2018</td>
+      <td>04/06/2018</td>
+      <td>35</td>
+    </tr>
+    <tr>
+      <td>thehungryjpeg.com</td>
+      <td>12/03/2018</td>
+      <td>12/06/2018</td>
+      <td>19</td>
+    </tr>
+    <tr>
+      <td>www.klook.com</td>
+      <td>14/03/2018</td>
+      <td>12/06/2018</td>
+      <td>28</td>
+    </tr>
+    <tr>
+      <td>www.steinmart.com</td>
+      <td>15/03/2018</td>
+      <td>09/07/2018</td>
+      <td>12</td>
+    </tr>
+    <tr>
+      <td>www.marveloptics.com</td>
+      <td>28/03/2018</td>
+      <td>22/09/2018</td>
+      <td>15</td>
+    </tr>
+  </tbody>
+</table>
+<p class="img-caption">Table 1: Sites affected by webfotce.me attack.</p>
+
+Compared to Ticketmaster, the impact of the breach on these sites is much smaller. Correllations between the dates of infection indicate that these sites were probably infected via a shared third-party (i.e. Inbenta) which was compromised. This shows how hackers can quickly achieve much greater scale by going for third-party services whose scripts will be loaded on many different sites.
+
+### NewEgg
+
+Like British Airways, NewEgg were hit by a [targetted attack](https://www.riskiq.com/blog/labs/magecart-newegg/). In this case the collection server was specific for the target. The hackers registered `neweggstats.com` in order to have a legitimate looking domain so that they could avoid suspicion for as long as possible.
+
+Looking at our data, we see that pages on `secure.newegg.com` were sending requests to `neweggstats.com` for just over a month, between 15th August and 18th September, with 90 pages affected in our dataset.
```

Comment:
When you say "pages were affected", this is the number of distinct page loads which had this specific third-party loaded right? Or 90 different URLs?